command/snapshot/inspect/formatter.go

```diff
@@ -57,9 +57,22 @@ func (_ *prettyFormatter) Format(info *OutputFormat) (string, error) {
 	fmt.Fprintf(tw, "\n %s\t%s\t%s\t", "----", "----", "----")
 	fmt.Fprintf(tw, "\n Total\t\t%s\t", ByteSize(uint64(info.TotalSize)))
 
+	if len(info.StatsKV) > 0 {
```

Comment:
Instead of checking length only to ensure this condition but not to use that logic again we could do something like this:
```
if info.StatsKV != nil
```